### DISCUSSION TITLE
Authorize GitHub Desktop - Privacy Issues

### DISCUSSION THREAD

**Discussion by porkyfork** [community/community#108476] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Feature Area

Projects

### Body

I have installed Github Desktop, but am now beung asked to 'authorise', see below, what 'use data' is it reading ? my Profile Data, or other data hidden on my PCs 'User Data' folder.

Personal user data
Full access

This application will be able to read and write all user data. This includes the following:

    Private email addresses
    Private profile information
    Followers

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @porkyfork ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by steveward** [comment]
@porkyfork thanks for the question! The data mentioned here is specific to your GitHub account, not anything on your machine. See [here](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/authorizing-oauth-apps#types-of-requested-data) for an example of data that GitHub Desktop might use. GitHub Desktop's [EULA](https://desktop.github.com/eula/) also has information about data the app collects. Let me know if you have any questions about that.
